doc/descriptors.md

```diff
@@ -131,6 +131,27 @@ Key order does not matter for `sortedmulti()`. `sortedmulti()` behaves in the sa
 as `multi()` does but the keys are reordered in the resulting script such that they
 are lexicographically ordered as described in BIP67.
 
+#### Basic multisig example
+
+For a good example of a basic M-of-N multisig between multiple participants using descriptor
+wallets and PSBTs, as well as a signing flow, see [this functional test](/test/functional/wallet_multisig_descriptor_psbt.py).
+The basic steps are:
+
+  1. Every participant generates an xpub. The most straightforward way is to create a new wallet
```

Comment:
"is to create a new descriptor wallet. Avoid reusing this wallet for any other purpose."

Could add a hint here on how to extract the xpub. Using `listdescriptors` and maybe pick the one from `pkh`, since you're least likely to accidentally reuse that (legacy addresses).

Reply:
Just for my understanding, what are the practical drawbacks/risks of someone re-using this wallet for other purposes? To me it's not obvious why I shouldn't have a "signer" wallet that signs for a few multisigs and also has some of its own funds

Reply:
Ah OK got it - thanks!

Reply:
If you have a wallet with just signers, this may not matter, but generally, if you have one wallet with multiple unrelated funds/descriptors imported, its balances will be the sum of all, with no way to separate them out. That may not be what you want.